src/Compilers/CSharp/Portable/Binder/Semantics/AccessCheck.cs

```diff
@@ -221,6 +221,11 @@ private static bool IsNamedTypeAccessible(NamedTypeSymbol type, Symbol within, r
 
             failedThroughTypeCheck = false;
 
+            if (containingType.IsTupleType)
+            {
+                containingType = containingType.TupleUnderlyingType;
```

Comment:
I think we are fine here, this function doesn't report any accessibility diagnostics, it simply returns true/false.

---
In reply to: [62272451](https://github.com/dotnet/roslyn/pull/11136#discussion_r62272451) [](ancestors = 62272451,62271140)